.github/policies/resourceManagement.yml

```diff
@@ -290,6 +290,39 @@ configuration:
       then:
       - removeLabel:
           label: untriaged
+          
+    - description: Add breaking change doc instructions to issue
+      if:
+      - payloadType: Issues
+      - labelAdded:
+          label: breaking-change
+      then:
+      - addReply:
+          reply: >-
+            Refer to the [.NET SDK breaking change guidelines](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/breaking-change-guidelines.md#required-process-for-all-net-sdk-breaking-changes)
+      
+    - description: Add breaking change instructions to PR.
+      if:
+      - payloadType: Pull_Request
+      - labelAdded:
+          label: breaking-change
+      then:
+      - addLabel:
+          label: needs-breaking-change-doc-created
+      - addReply:
+          reply: >-
+            Added `needs-breaking-change-doc-created` label because this PR has the `breaking-change` label. 
+
+
+            When you commit this breaking change:
+
+
+            1. [ ] Create and link to this PR and the issue a matching issue in the dotnet/docs repo using the [breaking change documentation template](https://aka.ms/dotnet/docs/new-breaking-change-issue), then remove this `needs-breaking-change-doc-created` label.
+
+            2. [ ] Ask a committer to mail the `.NET SDK Breaking Change Notification` email list.

```

Comment:
Do we want to be doing this as well? Isn't that meant only for sdk repo breaking changes?

Reply:
It's meant for any change that is considered breaking at the .NET SDK level. I _expect_ this will be fairly low volume for us but if it gets too high we'll re-evaluate.

Reply:
> breaking at the .NET SDK level.

Not sure what this means actually, isn't every change breaking at the sdk level e.g., in the sense that like `dotnet build` will behave differently?

Anyway, should we apply the breaking-change label to every breaking change PR and ignore this message for non-sdk breaking changes, or avoid applying the label for non-sdk breaking changes altogether?

Reply:
There is some nuance to what we apply it to. Generally, it's only going to be used for cases where a compiler change is likely to impact a significant number of customers when they upgrade the .NET SDK only (not changing TFM, warning level, etc ...). For example, the change to make iterators clear on `Dispose` would likely fall into this category.

Typically we have 1-2 of these in a given release.